test/integration_test.go

```diff
@@ -1376,3 +1379,60 @@ func (s *IntegrationSuite) TestExplain() {
 		})
 	}
 }
+
+func (s *IntegrationSuite) TestSequence() {
+	var (
+		db = s.DB()
+		t  = s.T()
+	)
+
+	rt, err := runtime.Load("arana", "employees")
+	if err != nil {
+		panic(err)
+	}
+	ctx := context.WithValue(context.Background(), proto.RuntimeCtxKey{}, rt)
+	ctx = context.WithValue(ctx, proto.ContextKeyTenant{}, "arana")
+	ctx = context.WithValue(ctx, proto.ContextKeySchema{}, "employees")
+	_, err = proto.LoadSequenceManager().CreateSequence(ctx, "arana", "employees", proto.SequenceConfig{Name: "student", Type: "group"})
+	if err != nil {
+		panic(err)
+	}
+
+	type testCase struct {
+		sql       string
+		exceptVal int64
+	}
+
+	for _, it := range [...]testCase{
+		{
+			"select student.nextVal",
+			1,
+		},
+		{
+			"select student.currVal",
+			1,
+		},
+		{
+			"select student.nextVal",
+			2,
+		},
+		{
+			"select notexist.currVal",
```

Comment:
ditto. if the user executes sql like 'select student.currVal from xxx',  do we expect some error?

Reply:
I think there is no need to report an error, because there may be real query scenarios to query this field